Contributing-code-to-Oppia.md

```diff
@@ -65,21 +65,30 @@ Welcome! Please make sure to follow the instructions above if you haven't alread
 
 After that, you can choose a good first issue from the [list of good first issues](https://github.com/oppia/oppia/labels/good%20first%20issue). These issues are hand-picked to ensure that you don't run into unexpected roadblocks while working on them, and each of them should have clear instructions for new contributors. If you see one that doesn't, please let us know via [GitHub Discussions](https://github.com/oppia/oppia/discussions) and we'll fix it. For other issues, you might need to be more independent because we might not know how to solve them either.
 
-You can also browse good first issues for each of the core Oppia Web teams to find something you'd enjoy working on:
+You can also browse good first issues for each of the core Oppia Web teams to find something you'd enjoy working on. Please only choose issues that have **not yet** been assigned! Here are the project boards for the different teams:
```

Comment:
What about checkbox issues? They're assigned to the dev who monitors and assigns the checkboxes. This may mislead contributors to think they can't work on them.

Reply:
Good catch! Done.